app/_src/mesh/features/openshift-quickstart.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Get started with Red Hat OpenShift and Kong Mesh
+content_type: tutorial
+description: This guide explains how to get started on Kong Mesh with Red Hat OpenShift.
+---
+
+This is a quickstart to get you up and running with Kong Mesh in standalone mode on OpenShift.
+
+This tutorial does not require a license, Kong Mesh can start in evaluation mode, and limits you 5 dataplanes/sidecars to use. Just enough dataplanes to get comfortable with the product.
+
+For OpenShfit, we'll be spinning up a ROSA 4.12 cluster. This tutorial does also assume some base level knowledge of OpenShift.
+
+This tutorial will cover:
+
+* How to use the Red Hat Certified Kong Mesh Images
+* How to implement the required OpenShift SecurityContextConstraints (SCC) for the kong-mesh sidecar
+* Deploy and join the Kong for Kubernetes Ingress Controller (KIC) to the mesh
+* Deploy a sample application, bookinfo, on the mesh and validate it's all working
+
+## Prerequisites
+* [ROSA CLI installed](https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-installing-rosa.html) or another OpenShift 4.12 cluster with the ability to create LoadBalancer type Kubernetes Services
+* [kubectl CLI installed](https://kubernetes.io/docs/tasks/tools/)
+* [OpenShift CLI installed](https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/getting-started-cli.html)
+* [Helm 3 installed](https://helm.sh/docs/intro/install/)
+
+## Install ROSA
+
+1. Create System Variables:
+    ```bash
+    CLUSTER_NAME=kong-mesh-2
+    REGION=us-west-2
+    ```
+
+1. Create a small ROSA cluster:
+    ```bash
+    rosa create cluster --cluster-name=$CLUSTER_NAME --region=$REGION --multi-az=false --version 4.12.13
+    ```
+
+1. When the Cluster install is complete, create a cluster-admin user:
+    ```bash
+    rosa create admin --cluster $CLUSTER_NAME
+    ```
+
+1. Validate you can login to the cluster via the credentials provided by the rosa cli stdout. Once login is successful you can proceed to the next step.
+
+## Install Kong Mesh in Standalone Mode
```

Comment:
Why do we install in standalone mode instead of a different mode?

Reply:
for simplicity, but converting the instructions to a zone, or mesh manager zone is more appropriate